components/nexusoperations/frontend/handler.go

```diff
@@ -118,8 +119,10 @@ func (h *completionHandler) CompleteOperation(ctx context.Context, r *nexus.Comp
 			metrics.NamespaceTag(nsName),
 		),
 		requestStartTime: startTime,
+		metricTagConfig:  h.Config.MetricTagConfig,
 	}
 	ctx = rCtx.augmentContext(ctx, r.HTTPRequest.Header)
+	rCtx.enrichNexusOperationMetrics(r.HTTPRequest.Method, r.HTTPRequest.URL.Path, r.HTTPRequest.Header)
```

Comment:
Those aren't service and operation names. These labels can't be available on this metric.